FindTown/FindTown/Presentation/HomeScene/Home/SubViews/FilterCollectionViewCell.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  CategoryCollectionViewCell.swift
+//  FindTown
+//
+//  Created by 김성훈 on 2023/01/13.
+//
+
+import UIKit
+
+import FindTownUI
+import RxSwift
+
+final class FillterCollectionViewCell: UICollectionViewCell {
+
+    // MARK: Properties
+    
+    static var reuseIdentifier: String {
+        return String(describing: self)
+    }
+    
+    private var disposeBag = DisposeBag()
+    
+    // MARK: Views
+    
+    private let searchCategoryBtn = FTButton(style: .buttonFilter)
+    
+    // MARK: Life Cycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        contentView.addSubview(searchCategoryBtn)
+        
+        searchCategoryBtn.isUserInteractionEnabled = false
+        
+        NSLayoutConstraint.activate([
+            searchCategoryBtn.topAnchor.constraint(equalTo: contentView.topAnchor),
+            searchCategoryBtn.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            searchCategoryBtn.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            searchCategoryBtn.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+        ])
+        
+        searchCategoryBtn.configuration?.contentInsets.leading = 12
+        searchCategoryBtn.configuration?.contentInsets.trailing = 12
```

Comment:
```suggestion
        searchCategoryBtn.configuration?.contentInsets = NSDirectionalEdgeInsets(top: 0.0, leading: 12.0, bottom: 0.0,trailing: 12.0)
```
이렇게 한 줄로 작성하면 조금 더 간결하지 않을까 싶습니다..!

Reply:
좋아요 좋아요 !!